plugins/inventory/terraform_state.py

```diff
@@ -10,10 +10,11 @@
   - Aubin Bikouo (@abikouo)
 short_description: Builds an inventory from resources created by cloud providers.
 description:
-  - This plugin works with an existing state file to create an inventory from resources created by cloud providers.
-  - The plugin accepts a Terraform backend config to an existing state file or a path to an existing state file.
-  - Uses a YAML configuration file that ends with terraform_state.(yml|yaml).
-  - To read the state file command ``Terraform show`` is used.
+  - This plugin works by making a temporary directory, writing a C(main.tf) file with the configured backend, then running C(terraform init),
+    followed by C(terraform show) in JSON format to read state.
+  - The plugin accepts a Terraform backend config to an existing backend, or path(s) to files containing backend config.
+  - Uses a YAML configuration file that ends with C(terraform_state.yml) or C(terraform_state.yaml).  For example, C(aws_terraform_state.yaml).
+  - Currently only supports the following Terraform providers - C(aws), C(azurerm), C(google).
```

Comment:
#146 would add the ability to use any provider.

Reply:
Ok, nice, but again, until such time, the text I've suggested above would have saved me a few hours to realise the module does nothing for me when using a provider not in that currently supported group.